pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/JUnitAssertionsShouldIncludeMessage.xml

```diff
@@ -426,4 +426,18 @@ public class TestNgTestCase {
 }
         ]]></code>
     </test-code>
+    <test-code>
+        <description><![CDATA[
+Issue #1009: JUnitAssertionsShouldIncludeMessage - False positive with assertEquals
+     ]]></description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+import junit.framework.TestCase;
```

Comment:
I'd prefer to have here a actual JUnit5 example - the TestCase class is actually Junit3.

Just use the code snipped from the issue:

```java
package test.java;

import static org.junit.jupiter.api.Assertions.assertEquals;
import org.junit.jupiter.api.Test;

class SimpleTest {
    @Test
    public void simpleMethod() {
        assertEquals(0, 1, "Not equals 0 != 1");
    }
}
```

Reply:
Done.